guide/english/javascript/standard-objects/string/string-fromcodepoint/index.md

```diff
@@ -1,15 +1,49 @@
 ---
-title: String Fromcodepoint
+title: String FromCodePoint
 ---
-## String Fromcodepoint
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/standard-objects/string/string-fromcodepoint/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+`String.fromCodePoint` returns a string from a list or sequence of code points.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+## Syntax
+
+```javascript
+String.fromCodePoint(num1[, ...[, numN]])
+```
+
+### Parameters
+
+>**num1, ..., numN**
+
+A sequence of code points.
+
+## Description
```

Comment:
Also I think that it is better if the description was before the syntax

Reply:
Done.